cicd/pipelines/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="using-tekton-chains-for-openshift-pipelines-supply-chain-security"]
+= Using Tekton Chains for OpenShift Pipelines supply chain security
+include::_attributes/common-attributes.adoc[]
+:context: using-tekton-chains-for-openshift-pipelines-supply-chain-security
+
+toc::[]
+
+:FeatureName: Tekton Chains
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+Tekton Chains is a Kubernetes Custom Resource Definition (CRD) controller. You can use it to manage the supply chain security of the tasks and pipelines created using {pipelines-title}.
+
+By default, Tekton Chains observes all task run executions in your {product-title} cluster. When the task runs complete, Tekton Chains takes a snapshot of the task runs. It then  converts the snapshot to one or more standard payload formats, and finally signs and stores all artifacts. 
+
+Tekton Chains uses the `Result` and `PipelineResource` objects, and in their absence the URLs and qualified digests of the OCI images, to capture information about task runs.
```

Comment:
It took me a couple times to read this sentence. What do you think about this rephrase? Do you think it keeps the original meaning while being easier to read? 

`To capture information about task runs, Tekton Chains uses the Result and PipelineReource objects or the URLs and qualified digests of the OCI images.`